kahuna/public/js/preview/image.html

```diff
@@ -46,6 +46,13 @@
 
     <div class="preview__info" ng:if="! ctrl.hideInfo">
         <div class="flex-container">
+            <span class="preview__info__collection"
+                ng-repeat="collection in ctrl.image.data.collections"
+                gr:tooltip="{{collection.data.path.join(' ▸ ')}}">
```

Comment:
Might also be easier to use ng-hover though? Will mock something up.

Akash Askoolum
Software Developer | Digital Development | Editorial Tools

(sent from mobile)
On 23 Dec 2015 17:00, "James Gorrie" <notifications@github.com> wrote:

> In kahuna/public/js/preview/image.html
> <https://github.com/guardian/grid/pull/1579#discussion_r48361629>:
>
> > @@ -46,6 +46,13 @@
> >
> >      <div class="preview__info" ng:if="! ctrl.hideInfo">
> >          <div class="flex-container">
> > +            <span class="preview__info__collection"
> > +                ng-repeat="collection in ctrl.image.data.collections"
> > +                gr:tooltip="{{collection.data.path.join(' ▸ ')}}">
>
> We could actually create the template in the controller and just pass it
> through?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/grid/pull/1579/files#r48361629>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396